common/djangoapps/student/tests/test_models.py

```diff
@@ -258,7 +259,7 @@ class UserCelebrationTests(SharedModuleStoreTestCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.course = CourseFactory()
+        cls.course = CourseFactory(default_store=ModuleStoreEnum.Type.split)
```

Comment:
Centralizing logic in `courseware_mfe_is_active`, which contains a check that `not course_key.deprecated`, meant that all these tests broke because they were using Old Mongo test courses.